web-client/src/app.tsx

```diff
@@ -271,13 +277,11 @@ const app = {
 
     root.render(
       <Container app={cerebralApp}>
-        {!process.env.CI && (
-          <>
-            <IdleActivityMonitor />
-            <AppInstanceManager />
-            <GlobalModalWrapper />
-          </>
-        )}
```

Comment:
This check appears unnecessary--tests are not failing with it removed. It was originally added years ago, apparently to prevent accessibility checks from failing at that time. I think it is best to remove because 1) as said, it appears to be unnecessary and 2) it is a "gotcha" for anyone trying to write automated tests depending on the components within this block (as I was!).

In fact, I think this gotcah already _did_ trip us uo. See my comment on `broadcastLogoutAction.js`.